lib/zipper.ml

```diff
@@ -0,0 +1,57 @@
+(** This file contains the zipper implementation *)
+
+module Zipper : sig
```

Comment:
Why did you decide to remove the `.mli`?
It is more conventional to put the interface in the mli file than in the ml file in this way. It also avoids having a module `Zipper.Zipper`, which is not ideal!

Reply:
No problem! Your contributions are of high quality, and you are here to learn :slightly_smiling_face:

Reply:
Resolved